tools/js-sdk-release-tools/src/common/rushUtils.ts

```diff
@@ -70,27 +80,36 @@ export async function buildPackage(
     logger.info(`Start to build package in '${relativePackageDirectoryToSdkRoot}'.`);
 
     const { name } = await getNpmPackageInfo(relativePackageDirectoryToSdkRoot);
-    await updateRushJson({
-        packageName: name,
-        projectFolder: unixify(relativePackageDirectoryToSdkRoot),
-        versionPolicyName: options.versionPolicyName
-    });
-
-    logger.info(`Start to rush update.`);
-    await runCommand(`node`, [rushScript, 'update'], runCommandOptions, false);
-    logger.info(`Rush update successfully.`);
-
-    await migratePackage(packageDirectory);
-
-    logger.info(`Start to build package '${name}'.`);
-    await runCommand('node', [rushScript, 'build', '-t', name, '--verbose'], runCommandOptions);
+    if (isRushRepo(options.sdkRepoRoot)){
+        await updateRushJson({
+            packageName: name,
+            projectFolder: unixify(relativePackageDirectoryToSdkRoot),
+            versionPolicyName: options.versionPolicyName
+        });
+
+        logger.info(`Start to rush update.`);
+        await runCommand(`node`, [rushScript, 'update'], runCommandOptions, false);
+        logger.info(`Rush update successfully.`);
+
+        await migratePackage(options.sdkRepoRoot, packageDirectory);
+        logger.info(`Start to build package '${name}'.`);
+        await runCommand('node', [rushScript, 'build', '-t', name, '--verbose'], runCommandOptions);
+    } else {
```

Comment:
Will you check pnpm workspace file exist before continue

Reply:
I think this is unnecessary. Currently, we are just adding a condition to decide whether to use `rush` or `pnpm`. Moreover, we plan to remove `rush` in the future and keep only `pnpm`.